shared/src/main/kotlin/com/egm/stellio/shared/model/ErrorResponse.kt

```diff
@@ -108,5 +115,6 @@ enum class ErrorType(val type: URI) {
     ACCESS_DENIED(URI("https://uri.etsi.org/ngsi-ld/errors/AccessDenied")),
     NOT_IMPLEMENTED(URI("https://uri.etsi.org/ngsi-ld/errors/NotImplemented")),
     UNSUPPORTED_MEDIA_TYPE(URI("https://uri.etsi.org/ngsi-ld/errors/UnsupportedMediaType")),
-    NOT_ACCEPTABLE(URI("https://uri.etsi.org/ngsi-ld/errors/NotAcceptable"))
+    NOT_ACCEPTABLE(URI("https://uri.etsi.org/ngsi-ld/errors/NotAcceptable")),
+    NONEXISTENT_TENANT(URI("https://uri.etsi.org/ngsi-ld/errors/NonexistentTenant"))
```

Comment:
NonExistentTenant*

Reply:
Sorry I saw it was in the spec. I just found it weird

Reply:
yes, in English, it is one word: https://dictionary.cambridge.org/dictionary/english/nonexistent